crates/db_schema/src/aggregates/community_aggregates.rs

```diff
@@ -1,20 +1,36 @@
 use crate::{
   aggregates::structs::CommunityAggregates,
   newtypes::CommunityId,
-  schema::community_aggregates,
+  schema::{
+    community_aggregates,
+    community_aggregates::{community_id, subscribers},
+  },
   utils::{get_conn, DbPool},
 };
 use diesel::{result::Error, ExpressionMethods, QueryDsl};
 use diesel_async::RunQueryDsl;
 
 impl CommunityAggregates {
-  pub async fn read(pool: &mut DbPool<'_>, community_id: CommunityId) -> Result<Self, Error> {
+  pub async fn read(pool: &mut DbPool<'_>, for_community_id: CommunityId) -> Result<Self, Error> {
     let conn = &mut get_conn(pool).await?;
     community_aggregates::table
-      .filter(community_aggregates::community_id.eq(community_id))
+      .filter(community_id.eq(for_community_id))
       .first::<Self>(conn)
       .await
   }
+
+  pub async fn update_followers(
```

Comment:
Might be good to rename this `update_federated/non_local_followers`, and add a comment that this should only be done for federated communities.

Reply:
Makes sense, fixed.